expressEngine/logger/logger.js

```diff
@@ -0,0 +1,89 @@
+import { createLogger, format, transports } from 'winston';
+import moment from 'moment';
+
+const {
+  combine, colorize, simple,
+} = format;
+
+const getDetailsFromFile = (fileDetails) => {
```

Comment:
please try to add comments for this kind of functions, it's very unclear why it is required to split and replace